blackboardvc-portlet-webapp/src/main/webapp/WEB-INF/jsp/viewSession.jsp

```diff
@@ -97,6 +97,8 @@
 		<div class="col-md-12">
 			<div class="session-large-heading">Invite participants to your web conference</div>
 			<p class="session-descriptive-text">There are 2 ways to invite participants to your web conference.</p>
+			<br/>
+      <span class="session-descriptive-text"><spring:message code="telephonyWarning" text="telephonyWarning" /></span>
```

Comment:
Worth having better default `test`? Maybe just ` ` as the `text` so that if message code `telephonyWarning` is not present, gracefully degrades to one character of whitespace?